packages/sanes-chrome-extension/src/routes/restore-account/components/SetMnemonicForm.tsx

```diff
@@ -33,7 +41,7 @@ const SetMnemonicForm = ({ onSetMnemonic, onBack }: Props): JSX.Element => {
 
   //TODO optimize update of validators with array of dependencies
   const validator = useMemo(() => {
```

Comment:
If we will move this validator out of component function we can get rid of `useMemo` function,

Reply:
Like this? https://github.com/iov-one/ponferrada/pull/420/commits/e03e3202663a1115901c3aa8fb18c96ad82b00df

Reply:
yes